sound/voice_data.h

```diff
@@ -35,7 +34,11 @@ const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0
+#else // Male Voice
+const uint8_t spBATTLEVCURR[] PROGMEM = {0xc8,0x6d,0x46,0xd9,0x33,0xdc,0x10,0xb7,0x37,0x36,0x57,0x4b,0xc3,0xea,0x62,0x45,0xb3,0xa2,0x49,0xb1,0x8a,0x52,0xd7,0x29,0x79,0xc3,0x2a,0x46,0x9c,0x4f,0xed,0x2c,0xbb,0x38,0x31,0x3e,0xb7,0x3b,0xec,0xa4,0x2d,0x65,0xd3,0x4a,0x72,0xa2,0xb4,0xa1,0x49,0xc9,0xe6,0xf3,0xb2,0xa2,0x3c,0xe4,0xa4,0xbf,0x1b,0x55,0xab,0x90,0xb3,0x9c,0x6a,0xc4,0x73,0x42,0xf6,0x71,0x8a,0xd6,0xce,0x11,0x25,0xc7,0xc9,0xd6,0x3b,0x43,0xdc,0x1c,0xa7,0xe8,0xea,0x30,0x51,0x7a,0x9c,0x62,0xa3,0xa2,0x44,0xce,0x71,0x8a,0x8f,0xcc,0x12,0x39,0xc7,0xce,0xa6,0xca,0x42,0x95,0x2e,0x27,0x99,0x48,0x6b,0x53,0x32,0x9c,0xac,0x3d,0xbc,0x53,0x4e,0x0b,0xaa,0x36,0xd3,0xcc,0xc6,0x2d,0x4a,0x5e,0xad,0x33,0xab,0x84,0xc4,0x27,0xf1,0x76,0x4f,0xec,0x32,0x53,0xb9,0xdd,0x32,0x76,0xca,0xa2,0xa7,0x8e,0x68,0xd9,0x2d,0xcb,0x86,0xb2,0x2b,0x2d,0x8f,0xac,0x08,0xf6,0x99,0x90,0x5c,0xb2,0xc4,0xa4,0x76,0x4c,0x52,0xc9,0x32,0xe3,0xac,0x09,0x49,0x29,0xf3,0xa4,0xd3,0xb3,0x62,0x85,0xc4,0xf3,0x65,0xeb,0x8c,0x54,0x62,0xaf,0x27,0x6c,0x4a,0xce,0x88,0xb3,0xca,0xd0,0x4d,0xca,0x23,0xce,0x21,0xcc,0xd7,0x44,0x8f,0x24,0xbb,0x74,0xeb,0x92,0xd2,0x92,0xe8,0xd2,0x34,0x4b,0xaa,0xc9,0xbc,0x29,0x17,0x4f,0xdb,0x2e,0x8d,0x7a,0x02,0xb5,0x2d,0xa7,0x24,0xda,0x0b,0xb0,0x36,0x9d,0x92,0x68,0xce,0xc0,0xc7,0x72,0x48,0x82,0x3d,0x83,0x1a,0xdb,0xe6,0x90,0xf1,0x94,0xae,0x24,0x9b,0xc3,0xab,0x29,0xde,0x90,0xe2,0x8e,0xc2,0x2d,0xf4,0x4a,0x72,0x0a,0x92,0xb0,0x8c,0x2b,0xc9,0x2d,0x48,0x5c,0x2b,0x3f,0xa8,0x94,0x20,0x70,0xeb,0xbc,0x50,0x9a,0x82,0xc0,0xbd,0x6a,0x42,0x6a,0x8a,0xa7,0x0e,0x61,0x8f,0x38,0x06,0x58,0x7e,0x42,0x01,0xd3,0x75,0x10,0xbf,0x5b,0x63,0x29,0x4d,0x63,0xbc,0xb6,0xd2,0x2d,0xd5,0x4e,0x7a,0xda,0x2a,0xb7,0xd4,0x38,0xe1,0x6d,0x31,0x55,0xd3,0x9a,0x04,0xa5,0xd6,0x52,0x35,0x4f,0x92,0xe4,0x54,0x28,0xd3,0xd3,0x4c,0x91,0x73,0xc3,0x32,0x2f,0xa9,0x45,0x49,0x95,0xc7,0x35,0xa5,0x14,0x25,0x25,0x1a,0xab,0xa2,0xdb,0xf4,0x14,0x69,0x3c,0x5b,0x76,0x33,0x92,0xb7,0x6c,0x0b,0xca,0xc3,0x2c,0x22,0x35,0xdb,0x25,0x37,0x2b,0xf3,0xb0,0x18,0xb7,0x65,0x9c,0x28,0x5d,0xaa,0xcd,0xb6,0xf2,0x23,0x2f,0xad,0xf2,0xd8,0x2a,0x16,0xad,0x2d,0xd2,0x22,0xb9,0x48,0xc5,0x55,0x1f,0xb7,0x55,0xe2,0x4c,0x33,0x65,0x52,0xc9,0x88,0xbb,0x30,0xe5,0x0e,0xd9,0x23,0x69,0x42,0xd4,0x3b,0x2d,0xb7,0xb4,0x72,0x8a,0x19,0x93,0x9c,0xb2,0xc2,0xa8,0xae,0x85,0x4e,0xc8,0x0b,0xa3,0xbe,0x62,0xca,0x2e,0xcf,0x94,0x67,0x93,0x25,0x99,0x3c,0x51,0x9e,0x4d,0x96,0x64,0xf2,0xc2,0xb0,0x2f,0x45,0xb2,0xc8,0x33,0xc5,0xbe,0x40,0xc9,0x24,0x8f,0x84,0x6a,0x43,0x2c,0xc3,0x03};
```

Comment:
The description doesn't mention any changes to spBATTLEVELCURR, so why was it changed?

Reply:
Updated. Was a placeholder copy/paste. Was going to add male version. Aborted, deleted.

Reply:
Same for BATTERYLVL removal, it was just unused. I suppose the Female BATTLVLCURR could also be removed, but removing unused is unnecessary anyway.

Reply:
fixed.